debian/control

```diff
@@ -0,0 +1,30 @@
+Source: revpi-sos-report
+Section: electronics
+Priority: optional
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Homepage: https://github.com/RevolutionPi/revpi-sos-report
+Vcs-Browser: https://github.com/RevolutionPi/revpi-sos-report/tree/raspios/bullseye
+Vcs-Git: https://github.com/RevolutionPi/revpi-sos-report.git -b raspios/bullseye
+Rules-Requires-Root: no
+Build-Depends:
+  debhelper-compat (= 13),
+  dh-python,
+  python3-dev,
```

Comment:
```suggestion
  python3-all,
```
Because we provide public modules, we use -all. -dev is for build python extensions, only. This will bring the header files and so on.